lua/neotest-mocha/init.lua

```diff
@@ -55,28 +65,28 @@ function Adapter.discover_positions(file_path)
     ; Matches: `describe('context')`
     ((call_expression
       function: (identifier) @func_name (#any-of? @func_name "describe" "context")
-      arguments: (arguments (string (string_fragment) @namespace.name) (arrow_function))
+      arguments: (arguments (string (string_fragment) @namespace.name) (_))
```

Comment:
Hey @adrigzr What does `_` here represent? Could you kindly point me to some docs?

Reply:
It is the [wildcard node](https://tree-sitter.github.io/tree-sitter/using-parsers#wildcard-node):

> A wildcard node is represented with an underscore (_), it matches any node. This is similar to . in regular expressions. There are two types, (_) will match any named node, and _ will match any named or anonymous node.